src/screens/DiscoveryScreenV2/elements/RecentSearchItem.js

```diff
@@ -49,7 +49,7 @@ const styles = StyleSheet.create({
     alignSelf: 'center',
     paddingTop: 6,
     paddingBottom: 6,
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.white2` to `COLORS.white`. Make sure this change does not negatively impact the readability of the text on its background.
```diff
- color: COLORS.white2
+ color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->